tests/app/test_logging.py

```diff
@@ -24,10 +25,10 @@ def test_logging_message_endpoint(self):
         """logging message endpoint"""
         out = StringIO()
         sys.stdout = out
-        message = {'msg_to': 'richard', 'msg_from': 'torrance', 'subject': 'hello', 'body': 'hello world',
-                   'thread_id': ''}
+        message = {'msg_to': [constants.NON_SPECIFIC_INTERNAL_USER], 'msg_from': 'Gemma', 'subject': 'hello', 'body': 'hello world',
+                   'thread_id': '', 'ru_id': 'f1a5e99c-8edf-489a-9c72-6cabe6c387fc', 'survey': 'cb8accda-6118-4d3b-85a3-149e28960c54'}
```

Comment:
Are these values arbitrarily chosen or are they based on other data? Might be worth defining them as constants (at least within the file) so we know if the values themselves are important to the test, or whether just a non-None value is acceptable.

Reply:
Certain values would make the test fail when strict was set to true. I had to change the `msg_to` and `msg_from` values because it kept displaying an error message about how I didn't have the permission to enter `Richard` and `Torrance` as values. I had to try a whole bunch of different values (which I got from looking at what values were passed into other tests) until I found something that worked. `thread_id` and `survey` simply needed a valid non-none value.

Reply:
They're probably specific values that get set up as part of test setup. May be worth making them constants then otherwise someone will change them and wonder why the tests start failing.

Reply:
Which values do you want me to set as constants?

Reply:
I'd say probably the names (Gemma etc), the `ru_id` and `survey` uuids